src/GovUk.Education.ExploreEducationStatistics.Admin/Models/DataFileInfo.cs

```diff
@@ -9,17 +10,19 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Models
     public class DataFileInfo : FileInfo
     {
         [JsonConverter(typeof(EnumToEnumValueJsonConverter<FileType>))]
-        public new readonly FileType Type = FileType.Data;
+        public new const FileType Type = FileType.Data;
```

Comment:
Why the need for the `new` keyword here, I wonder?  Maybe you don't need it with `const`?

Reply:
Without `new`, we get this compiler warning:

> The keyword 'new' is required on 'Type' because it hides property 'FileType GovUk.Education.ExploreEducationStatistics.Common.Model.FileInfo.Type'

Given it's the same signature as the base class, you'd think it would be fine as it's not really hiding anything, but hey, I don't make the compiler rules :man_shrugging: 

I didn't really need to change it to `const` either, but I originally got the suggestion in another case that was similar to this. Seemed to make sense so thought I'd throw it in here too!

Reply:
Yeaqh I would've thought it would have been fine too, but meh!  Cool thanks.